gix-config/fuzz/fuzz_targets/fuzz_file_corpus_builder.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -eox pipefail
+
+CWD=$(pwd)
+
+ROOT=$1
+OUTPUT_CORPUS=$2
+FIXTURES_DIR=$(readlink -f $ROOT/gix-config/tests/fixtures)
+
+echo $ROOT
+echo $FIXTURES_DIR
+find $FIXTURES_DIR -name "*.config" -exec zip -j $OUTPUT_CORPUS {} \;
+
+# Generate configs.
+REPO=/tmp/config_repo
```

Comment:
Would you be able to use `mktemp` or the likes to get a temp directory that is more unique? I can imagine this to work on CI where the state is fresh, but would expect this not to be idempotent otherwise.

Reply:
I'm admit I'm not familiar with mktemp. But I'll look into it.